Gem/Transfer/ViewsModels/PriceImpactViewModel.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+import Localization
+
+enum PriceImpactValue: Equatable {
+    case none
+    case low(String)
+    case medium(String)
+    case high(String)
+    case positive(String)
+}
+
+struct PriceImpactViewModel {
+    let fromAssetData: AssetData
+    let toAssetData: AssetData
+    
+    var priceImpact: String { Localized.Swap.priceImpact }
```

Comment:
```suggestion
    var priceImpact: String { Localized.Swap.priceImpact }
```

I would keep this as part of `SwapScene` where it's presented

Reply:
Price impact title presented from PriceImpactView, its separated View with own viewModel